pages/common/fdroidcl.md

```diff
@@ -14,3 +14,11 @@
 - Download an APK file:
 
 `fdroidcl download {{app_id}}`
+
+- Search for an app in the index:
+
+`fdroidcl search {{regex}}`
+
+- Install an app to a connected device:
```

Comment:
Really minor thing, but perhaps this will read better?
```suggestion
- Install an app on a connected device:
```

Otherwise, looks good :D